sc/social/like/upgrades/v3047/configure.zcml

```diff
@@ -0,0 +1,17 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup">
+
+  <genericsetup:upgradeSteps
+      source="3046"
+      destination="3047"
+      profile="sc.social.like:default">
+
+     <genericsetup:upgradeStep
+        title="Adds new field to the control panel."
+        description="This fields will be used to generate 'og:image' when content not image."
+        handler=".update_registry"
+        />
```

Comment:
you can do this without a handler, take a look at this PR https://github.com/collective/sc.social.like/pull/136/files#diff-8101f2cdf8c9e74b0408e13d6e1eed44R10

Reply:
Ok